packages/uploadthing/src/next/core/approuter.ts

```diff
@@ -53,3 +53,16 @@ export const createNextRouteHandler = <TRouter extends FileRouter>(
 
   return { GET, POST };
 };
+
```

Comment:
Is this the best place to export this from? 🤔

Reply:
technically, since both solid & app router are based on Request / Response api, yes, either that or make a new file called `node-handler.ts`

Reply:
these work on edge so node isn't the best name.

Reply:
changed it to `./src/internal/edge`